### PR TITLE
show bean properties when live process is connected

### PR DESCRIPTION
**Pull request by Eskibear**
for feature request  #181

### CODE REVIEW

**Comment by Eskibear** [comment]
Current impl looks like below:

image

**Comment by Eskibear** [comment]
Now: 

show `scope` in description section, which requires some cost to fetch the data for all entries.

image

**Comment by Eskibear** [comment]
Another concern is, "whether it's defined in workspace" is also marked in description section, we may need to revisit whether it's crowded or misunderstanding. 

image

**Comment by Tanyatang0** [comment]
> Another concern is, "whether it's defined in workspace" is also marked in description section, we may need to revisit whether it's crowded or misunderstanding.
> 
> image
1.Can parentheses be removed? Are commas enough to complete the distinction between different attributes? How about semicolons?
2. Can just use the icon to distinguish whether it is defined or not?
image

**Comment by Eskibear** [comment]
per discussion, for simplicity we can remove `()`, just showing `singleton, defined` in description. `scope:` is crowded and for spring developers they have the knowledge of scope at the glance of `singleton`.

**Comment by Eskibear** [comment]
image

File: src/views/beans.ts [hunk 1]
```diff
@@ -65,29 +77,54 @@ class BeansDataProvider implements vscode.TreeDataProvider<TreeData> {
...
             item.collapsibleState = vscode.TreeItemCollapsibleState.None;
-
-            const themeColor = isLive ? new vscode.ThemeColor("charts.green") : undefined;
-            item.iconPath = new vscode.ThemeIcon("spring-bean", themeColor);
-
-            item.contextValue = isLive ? "spring:bean" : "spring:staticBean";
-
+            item.iconPath = new vscode.ThemeIcon("spring-bean", COLOR_LIVE);
+            item.contextValue = "spring:bean";
+            const commandOnClick = "spring.dashboard.bean.open";
```
Comment (jdneo) [inline src/views/beans.ts:90]: One question, we don't need to check `isLive` now?

**Review by jdneo — APPROVED** [review]

**Pull request merged**